recipes/arabic_reshaper/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "arabic_reshaper" %}
+{% set version = "2.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 008d71e6f5523833d8f4c8b5fd487d8bbed444c7c9a06665695afdb5ef47f214
+
+build:
+  noarch: python
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - setuptools
+    - configparser
+    - future
+
+test:
+  imports:
+    - arabic_reshaper
```

Comment:
last comment:
   - it looks like the upstream package includes unit tests, if you know how to call them please add them here. If not, just lets open a PR on the -feedstock after it is merged.

Reply:
If I understand correctly, one would simply have to run `python arabic_reshaper/tests/test_001_initialization.py` and `python arabic_reshaper/tests/test_002_reshaping.py` from the source directory.

The call to `setup()` in `setup.py` contains a `test_suite` kwarg; however, the test scripts are _not_ part of the distribution, so I don't think there's a way to run those tests on the built `conda` package.